.bit/responses/4.3-Week 4 Step 3.md

```diff
@@ -66,7 +66,7 @@ The [Fetch API](https://developer.mozilla.org/en-US/docs/Web/API/Fetch_API/Using
 
 To start a request, call the special function `fetch()`:
 
-```javascript 
+```JavaScript 
```

Comment:
In the other code blocks it is js not sure what is correct without looking it up

Reply:
looks like js is the correct syntax highlighting for fences